package.json

```diff
@@ -32,8 +31,7 @@
     "prepare:dev": "is-ci || (husky install && npm run prepare-commit-hooks)",
     "prepare": "node -e \"process.exit(process.env.NODE_ENV === 'production' ? 0 : 1)\" || npm run prepare:dev",
     "remove:dist": "shx rm -rf dist",
-    "generate:api": "node scripts/generate-api.cjs",
-    "postinstall": "node scripts/autocomplete.cjs"
+    "generate:api": "node scripts/generate-api.cjs"
```

Comment:
should  we delete `scripts/autocomplete.cjs` file as well?

Reply:
Now it's not included in the build, but we can delete because it is not used anyway